features/netsocket/mbed_lib.json

```diff
@@ -2,6 +2,22 @@
     "name": "nsapi",
     "config": {
         "present": 1,
-        "default-stack": "LWIP"
+        "default-stack": "LWIP",
+        "dns-response-wait-time": {
+            "help": "How long the DNS translator waits for a reply from a server",
+            "value": 5000
+        },
+        "dns-total-retries": {
+            "help": "Number of total DNS query retries that the DNS translator makes",
+            "value": 3
+        },
+        "dns-retries": {
+            "help": "Number of DNS query retries that the DNS translator makes per server",
```

Comment:
Whichever you decide for the previous, seems like it might be best to have the two settings phrased the same. I think this needs a bit more explanation - `before moving on to the next server. Total retries/attempts is always limited by "dns-total-retries/attempts"`.

Reply:
Updated the description.